src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -10,6 +10,7 @@
     <AssemblyName>CoreNodeModels</AssemblyName>
     <OutputPath>$(OutputPath)\nodes\</OutputPath>
     <DocumentationFile>$(OutputPath)\CoreNodeModels.XML</DocumentationFile>
+    <UseWPF>true</UseWPF>
```

Comment:
I think this flag is supposed to be used in .net5 or above.
Also I do not think it is a good idea to bring in WPF/UI dependencies in a core project
That is why there is a CodeNodeModelsWpf project.

Reply:
Thank you for the comment. I will revert this change, but it was necessary for me to use some of the wpf functionality initially. Thank you!